doc/rbd/iscsi-initiators.rst

```diff
@@ -12,9 +12,16 @@ Configuring the iSCSI Initiators
 
     .. warning::
 
-        Applications that use SCSI persistent group reservations (PGR) and
-        SCSI 2 based reservations are not supported when exporting a RBD image
-        through more than one iSCSI gateway.
+        The Ceph iSCSI gateway does not support reservations in any scenario.
+        ``tcmu-runner`` does not support reservations.
+
+    .. warning::
+
+        The iSCSI code is in maintenance as of November 2022. This means that
+        it is no longer in development, it is feature complete, and it will not
```

Comment:
Let's strike the "feature complete" bit ;)
```suggestion
        it is no longer in active development and it will not
```

Reply:
Good call. I'll make this change.